tests/src/test/scala/common/Wsk.scala

```diff
@@ -71,6 +71,8 @@ import whisk.utils.retry
 
 case class WskProps(
     authKey: String = WhiskProperties.readAuthKey(WhiskProperties.getAuthFileForTesting),
+    cert: String = WhiskProperties.getProperty("openwhisk.home") + "/ansible/roles/nginx/files/openwhisk-client-cert.pem",
+    key: String = WhiskProperties.getProperty("openwhisk.home") + "/ansible/roles/nginx/files/openwhisk-client-key.pem",
```

Comment:
@rabbah , i don't find the method to return a relative path to whisk home.
i just find blow method may be can use
```
    /**
     * The path to the CLI directory.
     */
    public static String getCLIDir() {
        return whiskHome + "/bin";
    }

    /**
     * The path to the Go CLI executable.
     */
    public static String getCLIPath() {
        return getCLIDir() + "/wsk";
    }
```